superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterControl.tsx

```diff
@@ -0,0 +1,872 @@
+                    }
+                  />
+                </Col>
+              </Row>
+            )}
+          </Col>
+          <Col span={12}>
+            <div className="control-label">{t('END')}</div>
+            <Select
+              options={UNTIL_MODE_OPTIONS}
+              value={UNTIL_MODE_OPTIONS.filter(
+                option => option.value === untilMode,
+              )}
+              onChange={(option: any) =>
+                onCustomRangeChange('untilMode', option.value)
+              }
+            />
+            {untilMode === 'specific' && (
+              <Row>
+                <DatePicker
+                  showTime
+                  // @ts-ignore
+                  value={dttmToMoment(untilDatetime)}
+                  // @ts-ignore
+                  onChange={(datetime: Moment) =>
+                    onCustomRangeChange(
+                      'untilDatetime',
+                      datetime.format(MOMENT_FORMAT),
+                    )
+                  }
+                  allowClear={false}
+                />
+              </Row>
+            )}
+            {untilMode === 'relative' && (
+              <Row gutter={8}>
+                <Col span={10}>
+                  <InputNumber
+                    placeholder={t('Relative quantity')}
+                    value={untilGrainValue}
+                    min={1}
+                    defaultValue={1}
+                    onStep={value =>
+                      onCustomRangeChange('untilGrainValue', value || 1)
+                    }
+                  />
+                </Col>
+                <Col span={14}>
+                  <Select
+                    options={UNTIL_GRAIN_OPTIONS}
+                    value={UNTIL_GRAIN_OPTIONS.filter(
+                      option => option.value === untilGrain,
+                    )}
+                    onChange={(option: any) =>
+                      onCustomRangeChange('untilGrain', option.value)
+                    }
+                  />
+                </Col>
+              </Row>
+            )}
+          </Col>
+        </Row>
+        {sinceMode === 'relative' && untilMode === 'relative' && (
+          <>
+            <div className="control-label">{t('ANCHOR RELATIVE TO')}</div>
+            <Row align="middle">
+              <Col>
+                <Radio.Group
+                  onChange={onCustomRangeChangeAnchorMode}
+                  defaultValue="now"
+                  value={anchorMode}
+                >
+                  <Radio key="now" value="now">
+                    {t('NOW')}
+                  </Radio>
+                  <Radio key="specific" value="specific">
+                    {t('Date/Time')}
+                  </Radio>
+                </Radio.Group>
+              </Col>
+              {anchorMode !== 'now' && (
+                <Col>
+                  <DatePicker
+                    showTime
+                    // @ts-ignore
```

Comment:
Why is this ignored?

Reply:
The original Andt Datepicker had type issues, but now there is no need. I will remove ts-ignore comments